Dockerfile

```diff
@@ -0,0 +1,34 @@
+# 1: Build the exe
+FROM rust:1.56 AS builder
+
+# WORKDIR /usr/src
+
+# RUN USER=root cargo new agora
+
+# COPY Cargo.toml Cargo.lock /usr/src/agora/
+
+WORKDIR /usr/src/agora
+
+COPY . ./
+RUN cargo build --release
+
+FROM ubuntu AS final
+
+ARG APP=/usr/src/app
+ENV APP_USER=agora
+
+RUN useradd --system ${APP_USER}
+
+COPY --from=builder /usr/src/agora/target/release/agora ${APP}/bin/agora
+COPY static .
+COPY ./.lnd ${APP}/.lnd
```

Comment:
This fails on my machine, since I don't have a `.lnd` directory.

Reply:
I was thinking about different ways of doing this. Could the user mount the certificate and macaroon individually to known paths inside the container? I.e. mount the two files individually, instead of mounting the directory.

Reply:
I think we should use a docker-compose file to do this at runtime, yes

Reply:
@SachinMeier You can also mount volumes at runtime with just the docker run command:

```
docker run -v ~/.lnd:/lnd -e LND_DIR=/lnd ...
```

Reply:
Is it possible to mount individual files? It might be nice to just mount the needed files, instead of a directory which must have a specific layout, and which might contain things that you don't want to pass through to the container.

Reply:
@casey Yeah I think that's possible:

```
docker run -v <HOST_MACHINE_FILE_PATH>:<CONTAINER_FILE_PATH> ...
```

Reply:
I would recommend against having a docker-compose file unless it's absolutely necessary, which it shouldn't be for agora's use case. docker-compose files make deployment more complex, and it's possible to mount individual files with just the `docker run` command.